package.json

```diff
@@ -67,7 +74,7 @@
     "npm": ">=7"
   },
   "dependencies": {
-    "@primer/behaviors": "1.1.0",
+    "@primer/behaviors": "1.0.3",
```

Comment:
:)

Reply:
lol not sure how that happened. i did pull the latest next branch.